docs/api.md

```diff
@@ -206,17 +418,36 @@ metadata:
     appstudio.application: new-demo-app
     appstudio.environment: staging
 spec:
+  # Application is a reference to the Application resource (defined in the same namespace) that we are deploying as part of this SnapshotEnvironmentBinding.
   application: new-demo-app
+
+  # Environment is the environment resource (defined in the namespace) that the binding will deploy to.
   environment: staging
+
+	# Snapshot is the Snapshot resource (defined in the namespace) that contains the container image versions for the components of the Application.
```

Comment:
Not sure what this red patch implies. But I see that it is present on multiple lines in this block and also visible in the final markdown view.